tests/data/json/simple_mapping.json

```diff
@@ -0,0 +1,44 @@
+{
+  "mapping-collection": {
+    "uuid": "A0000000-0000-4000-8000-000000000050",
+    "metadata": {
+      "title": "Trestle test mapping-collection",
+      "last-modified": "2021-01-01T00:00:00.000+00:00",
+      "version": "2021-01-01",
+      "oscal-version": "1.0.0"
```

Comment:
I think it is been added only recently i.e. not in `1.0.0` ?

Reply:
changed to 1.0.4